### DISCUSSION TITLE
Meeting 2024-11-26

### DISCUSSION THREAD

**Discussion by adamgreig** [rust-embedded/wg#806] [General]
This is the agenda for the next meeting. Please add anything you'd like to discuss below!

* Meetings: Tuesday 8pm Europe/Berlin time
* [Join the Matrix chat](https://matrix.to/#/#rust-embedded:matrix.org)
* [Nominated issues](https://github.com/search?q=org%3Arust-embedded+label%3Anominated+is%3Aopen&type=Issues)
* [IRC/Matrix logs](https://libera.irclog.whitequark.org/rust-embedded/2024-11-26)

**Comment by jannic** [comment]
This ticket has a valid point and wasn't answered for months:
- https://github.com/rust-embedded/embedded-hal/issues/577

TL;DR: There's no good way to communicate UART-specific errors using the embedded-io traits.

Any suggestions?

**Reply by jannic** [reply]
We talked about this and I added a summary to the ticket, https://github.com/rust-embedded/embedded-hal/issues/577#issuecomment-2501841369

**Comment by romancardenas** [comment]
Any updates on https://github.com/rust-embedded/cortex-m/pull/560 ?
